### DISCUSSION TITLE
Page Author now showing up

### DISCUSSION THREAD

**Discussion by Ravencentric** [retypeapp/retype#522] [Q&A] [answered]
https://github.com/Snaacky/thewiki/blob/master/docs/tutorials/plex.md?plain=1
https://thewiki.moe/tutorials/plex/

![image](https://user-images.githubusercontent.com/78981416/230479178-e310be9e-89fa-4cbb-87f4-621c389da472.png)

```markdown
---
label: Plex
description: Modify the Plex Windows App to Include an Extended MPV Version
author:
  name: ! ThaUnknown_#3951
  link: http://github.com/ThaUnknown
  avatar: https://avatars.githubusercontent.com/u/6506529
---

# Why modify the player?

You get:
- Less buffering
- Better quality
- Dolby Vision support
- HDR10+ support
- No transcoding
````

**Comment by geoffreymcgill** [comment, marked as answer]
Two issues:

1. The [`author`](https://retype.com/configuration/page/#author) config is an array, so prefixing the individual author group with `-` is required. 
2. Wrap the author with `"` double-quotes as Markdown doesn't like that leading `!`. I'm not sure Markdown fails on the `!`, although I  assume it's being treated a negative operator, so the parsers assumes `! ThaUnknown_#3951` is going to be a `number` or a `boolean`, but wrapping with `"` double-quotes instructs the parser to treat as a `string`.

Please try the following revised sample:

```md
---
label: Plex
description: Modify the Plex Windows App to Include an Extended MPV Version
author:
  - name: "! ThaUnknown_#3951"
    link: http://github.com/ThaUnknown
    avatar: https://avatars.githubusercontent.com/u/6506529
---

# Why modify the player?

You get:
- Less buffering
- Better quality
- Dolby Vision support
- HDR10+ support
- No transcoding
```

Hope this helps.

**Reply by Ravencentric** [reply]
the issue was indeed "!"
```markdown
---
label: Plex
description: Modify the Plex Windows App to Include an Extended MPV Version
author:
  name: "! ThaUnknown_#3951"
  link: http://github.com/ThaUnknown
  avatar: https://avatars.githubusercontent.com/u/6506529
---
```
![image](https://user-images.githubusercontent.com/78981416/230481465-762dddd9-6d3b-48cb-a981-26bfc3e9bc5a.png)

Thanks alot!

**Reply by geoffreymcgill** [reply]
ah, yes. The `author` config can be a single item or an array. Both `author` and `authors` are supported.
